src/blocking_client.rs

```diff
@@ -0,0 +1,262 @@
+//! Minimal blocking Redis client implementation
+//!
+//! Provides a blocking connect and methods for issuing the supported commands.
+
+use bytes::Bytes;
+use std::time::Duration;
+use tokio::net::ToSocketAddrs;
+use tokio::runtime::Runtime;
+
+pub use crate::client::Message;
+
+/// Established connection with a Redis server.
+///
+/// Backed by a single `TcpStream`, `Client` provides basic network client
+/// functionality (no pooling, retrying, ...). Connections are established using
+/// the [`connect`](fn@connect) function.
+///
+/// Requests are issued using the various methods of `Client`.
+pub struct Client {
+    /// The asynchronous `Client`.
+    inner: crate::client::Client,
+
+    /// A `current_thread` runtime for executing operations on the asynchronous
+    /// client in a blocking manner.
```

Comment:
A short explanation of *why* a `current_thread` runtime is used here would be nice.